package.json

```diff
@@ -27,9 +27,9 @@
     "queryResource": "query",
     "okapiInterfaces": {
       "cancellation-reason-storage": "1.1",
-      "circulation": "3.4 4.0 5.0 6.0",
+      "circulation": "3.4 4.0 5.0 6.0 7.0",
```

Comment:
This change appears to include the changes for sending date and time for the two expiration date properties.

That format is not compatible with the older versions. How can this offer dual support, given that it is reliant upon the breaking changes for requests?

Reply:
@marc-johnson that's a good catch. I think we can only include `"circulation": "7.0"`